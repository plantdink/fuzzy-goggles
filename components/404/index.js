export * from './Component404';
export { default } from './Component404';
