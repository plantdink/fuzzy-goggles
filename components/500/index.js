export { default } from './Component500';
