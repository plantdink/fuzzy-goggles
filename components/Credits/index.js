export * from './Credits';
export { default } from './Credits';
