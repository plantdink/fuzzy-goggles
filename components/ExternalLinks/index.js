// export { default } from './ExternalLinks';
export * from './ExternalLinks';
