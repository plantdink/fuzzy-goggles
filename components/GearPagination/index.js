export * from './GearPagination';
export { default } from './GearPagination';
