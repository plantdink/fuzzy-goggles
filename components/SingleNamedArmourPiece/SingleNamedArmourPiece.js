import { gql } from '@apollo/client';
import SingleArmourPiece from '../SingleArmourPiece';

export const SINGLE_NAMED_ARMOUR_PIECE_QUERY = gql`
  query SINGLE_NAMED_ARMOUR_PIECE_QUERY($id: ID!) {
    allArmourTypes(where: { id: $id }) {
      id
      name
      piece
      modSlots
      coreAttributeTypeOne
      coreAttributeTypeTwo
      coreAttributeTypeThree
      maxCoreDamageLevel40
      maxCoreArmourLevel40
      maxCoreSkillTierLevel40
      maxCoreValueDamageLevel30
      maxCoreArmourLevel30
      maxCoreSkillTierLevel30
      offensiveAttributeOneLevel40
      offensiveAttributeTwoLevel40
      offensiveAttributeThreeLevel40
      offensiveAttributeFourLevel40
      offensiveAttributeOneMaxValueLevel40
      offensiveAttributeTwoMaxValueLevel40
      offensiveAttributeThreeMaxValueLevel40
      offensiveAttributeFourMaxValueLevel40
      offensiveAttributeOneMaxValueLevel30
      offensiveAttributeTwoMaxValueLevel30
      offensiveAttributeThreeMaxValueLevel30
      offensiveAttributeFourMaxValueLevel30
      defensiveAttributeOneLevel40
      defensiveAttributeTwoLevel40
      defensiveAttributeThreeLevel40
      defensiveAttributeFourLevel40
      defensiveAttributeOneMaxValueLevel40
      defensiveAttributeTwoMaxValueLevel40
      defensiveAttributeThreeMaxValueLevel40
      defensiveAttributeFourMaxValueLevel40
      defensiveAttributeOneMaxValueLevel30
      defensiveAttributeTwoMaxValueLevel30
      defensiveAttributeThreeMaxValueLevel30
      defensiveAttributeFourMaxValueLevel30
      utilityAttributeOneLevel40
      utilityAttributeTwoLevel40
      utilityAttributeThreeLevel40
      utilityAttributeFourLevel40
      utilityAttributeOneMaxValueLevel40
      utilityAttributeTwoMaxValueLevel40
      utilityAttributeThreeMaxValueLevel40
      utilityAttributeFourMaxValueLevel40
      utilityAttributeOneMaxValueLevel30
      utilityAttributeTwoMaxValueLevel30
      utilityAttributeThreeMaxValueLevel30
      utilityAttributeFourMaxValueLevel30
      uniqueAttribute
      notes
      flavourText
      availableWT5
      isNamed
      image {
        id
        altText
        image {
          publicUrlTransformed
        }
        altText
      }
      brand {
        id
        name
        image {
          id
          altText
          image {
            publicUrlTransformed
          }
        }
      }
      armourTalent {
        id
        name
        piece
        descriptionPVE
        isNamed
        isExotic
        image {
          id
          altText
          image {
            publicUrlTransformed
          }
        }
      }
    }
  }
`;

export default function SingleNamedArmourPiece({ singleNamedPiece }) {
  return (
    <>
      <SingleArmourPiece armourPiece={singleNamedPiece} />
    </>
  );
}
