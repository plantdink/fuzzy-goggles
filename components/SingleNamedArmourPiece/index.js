export * from './SingleNamedArmourPiece';
export { default } from './SingleNamedArmourPiece';
