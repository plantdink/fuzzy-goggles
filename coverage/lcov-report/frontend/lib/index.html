
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for frontend/lib</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../index.html">All files</a> frontend/lib</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">84.61% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>55/65</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">64% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>16/25</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">71.42% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>20/28</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">84.78% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>39/46</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
        <template id="filterTemplate">
            <div class="quiet">
                Filter:
                <input oninput="onInput()" type="search" id="fileSearch">
            </div>
        </template>
    </div>
    <div class='status-line high'></div>
    <div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file high" data-value="abbreviationsDictionary.js"><a href="abbreviationsDictionary.js.html">abbreviationsDictionary.js</a></td>
	<td data-value="100" class="pic high">
	<div class="chart"><div class="cover-fill cover-full" style="width: 100%"></div><div class="cover-empty" style="width: 0%"></div></div>
	</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="14" class="abs high">14/14</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="12" class="abs high">12/12</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="8" class="abs high">8/8</td>
	</tr>

<tr>
	<td class="file high" data-value="displayNumbers.js"><a href="displayNumbers.js.html">displayNumbers.js</a></td>
	<td data-value="90.9" class="pic high">
	<div class="chart"><div class="cover-fill" style="width: 90%"></div><div class="cover-empty" style="width: 10%"></div></div>
	</td>
	<td data-value="90.9" class="pct high">90.9%</td>
	<td data-value="11" class="abs high">10/11</td>
	<td data-value="44.44" class="pct low">44.44%</td>
	<td data-value="9" class="abs low">4/9</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="10" class="abs high">10/10</td>
	</tr>

<tr>
	<td class="file high" data-value="displayStrings.js"><a href="displayStrings.js.html">displayStrings.js</a></td>
	<td data-value="84.61" class="pic high">
	<div class="chart"><div class="cover-fill" style="width: 84%"></div><div class="cover-empty" style="width: 16%"></div></div>
	</td>
	<td data-value="84.61" class="pct high">84.61%</td>
	<td data-value="13" class="abs high">11/13</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="85.71" class="pct high">85.71%</td>
	<td data-value="7" class="abs high">6/7</td>
	<td data-value="83.33" class="pct high">83.33%</td>
	<td data-value="12" class="abs high">10/12</td>
	</tr>

<tr>
	<td class="file medium" data-value="testUtils.js"><a href="testUtils.js.html">testUtils.js</a></td>
	<td data-value="74.07" class="pic medium">
	<div class="chart"><div class="cover-fill" style="width: 74%"></div><div class="cover-empty" style="width: 26%"></div></div>
	</td>
	<td data-value="74.07" class="pct medium">74.07%</td>
	<td data-value="27" class="abs medium">20/27</td>
	<td data-value="0" class="pct low">0%</td>
	<td data-value="2" class="abs low">0/2</td>
	<td data-value="56.25" class="pct medium">56.25%</td>
	<td data-value="16" class="abs medium">9/16</td>
	<td data-value="68.75" class="pct medium">68.75%</td>
	<td data-value="16" class="abs medium">11/16</td>
	</tr>

</tbody>
</table>
</div>
                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank" rel="noopener noreferrer">istanbul</a>
                at Tue Apr 05 2022 07:32:49 GMT+1000 (Australian Eastern Standard Time)
            </div>
        <script src="../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../sorter.js"></script>
        <script src="../../block-navigation.js"></script>
    </body>
</html>
    