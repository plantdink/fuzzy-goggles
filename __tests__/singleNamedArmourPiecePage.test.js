import { render, screen, cleanup, waitFor } from '@testing-library/react';
import { MockedProvider } from '@apollo/react-testing';
import { fakeNamedArmourPieceQuery } from '../lib/testUtils';
import { SINGLE_NAMED_ARMOUR_PIECE_QUERY } from '../components/SingleNamedArmourPiece';
import SingleNamedArmourPiecePage from '../pages/gear/namedArmourPiece/[id]';

afterEach(cleanup);

const namedArmourPiece = fakeNamedArmourPieceQuery();

const mocks = [
  {
    request: {
      query: SINGLE_NAMED_ARMOUR_PIECE_QUERY,
      variables: {
        id: namedArmourPiece.id,
      },
    },
    result: {
      data: {
        allArmourTypes: [namedArmourPiece],
      },
    },
  },
];

const errorMocks = [
  {
    request: {
      query: SINGLE_NAMED_ARMOUR_PIECE_QUERY,
      variables: {
        id: namedArmourPiece.id,
      },
    },
    result: {
      errors: [
        {
          message: 'Named Armour Piece not found.',
        },
      ],
    },
  },
];

describe('<SingleGearsetPage />', () => {
  it('displays loading when the page first renders', () => {
    const { container, debug } = render(
      <>
        <MockedProvider mocks={mocks}>
          <SingleNamedArmourPiecePage
            query={{ id: `${namedArmourPiece.id}` }}
          />
        </MockedProvider>
      </>
    );

    const loaderSpinner = screen.getByTestId('tail-spin-loading');
    expect(loaderSpinner).toBeInTheDocument();
  });

  it('matches the snapshot', async () => {
    const { container, debug } = render(
      <>
        <MockedProvider mocks={mocks}>
          <SingleNamedArmourPiecePage
            query={{ id: `${namedArmourPiece.id}` }}
          />
        </MockedProvider>
      </>
    );

    await screen.findByTestId('singleYellowArmourPiece');
    expect(container).toMatchSnapshot();
  });

  it('loads the data and renders all components with data correctly for a single armour type page', async () => {
    const { container, debug } = render(
      <>
        <MockedProvider mocks={mocks}>
          <SingleNamedArmourPiecePage
            query={{ id: `${namedArmourPiece.id}` }}
          />
        </MockedProvider>
      </>
    );

    const testTitle = await screen.findByText(namedArmourPiece.name);
    expect(testTitle).toBeInTheDocument();

    const testSingleNamedArmourPieceContent = await screen.findByTestId(
      'singleYellowArmourPiece'
    );
    expect(testSingleNamedArmourPieceContent).toBeInTheDocument();
  });

  it('Throws an error when an item is not found', async () => {
    const { container, debug } = render(
      <>
        <MockedProvider mocks={errorMocks}>
          <SingleNamedArmourPiecePage
            query={{ id: `${namedArmourPiece.id}` }}
          />
        </MockedProvider>
      </>
    );

    expect(screen.queryByTestId('graphql-error')).toBeDefined();
    await waitFor(() => {});
  });
});
