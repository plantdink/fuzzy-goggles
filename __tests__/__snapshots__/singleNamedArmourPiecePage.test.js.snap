// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SingleGearsetPage /> matches the snapshot 1`] = `
<div>
  <div
    class="SingleArmourPiece__SingleGearItemStyle-sc-1lh23le-0 iZNXSJ"
  >
    <div
      class="single-gear-item__title-bar"
    >
      <h1
        class="single-gear-item__heading"
      >
        Punch Drunk
      </h1>
    </div>
    <img
      alt="Chest icon"
      class="single-gear-item__icon-image"
      data-testid="itemIcon"
      src="FakeArmourPiece.jpg"
    />
    <div>
      <p
        data-testid="itemPiece"
      >
        Mask
      </p>
      <p
        data-testid="itemNotes"
      >
        This mask has been manufactured by Douglas & Harding and comes with Weapon Damage as the primary Core attribute. It has one fixed minor attribute - 20% Headshot Damage - and one random minor attribute which can be any of the stats listed below. It also has one gear mod slot.
      </p>
    </div>
    <div
      class="single-gear-item__content"
      data-testid="singleArmourPiece"
    >
      <div
        class="single-gear-item__details"
      >
        <table
          data-testid="singleYellowArmourPiece"
        >
          <thead>
            <tr
              class="single-gear-item__title-bar"
            >
              <th
                class="single-gear-item__subheading"
              >
                Level 40 Stats
              </th>
            </tr>
            <tr
              class="single-gear-item__title-bar"
            >
              <th
                class="single-gear-item__subheading"
              >
                Core Attributes
              </th>
            </tr>
            <tr>
              <td />
              <td>
                Max Value
              </td>
            </tr>
          </thead>
          <tbody>
            <tr
              data-testid="itemCoreOneType"
            >
              <td>
                Weapon Damage
              </td>
              <td>
                +
                 
                15
                %
              </td>
            </tr>
          </tbody>
        </table>
        <table>
          <thead
            class="single-gear-item__title-bar"
          >
            <tr>
              <th
                class="single-gear-item__h3subheading"
              >
                Offensive Minor Attributes
              </th>
            </tr>
          </thead>
          <tbody>
            <tr
              data-testid="itemOffAttrOneMax40Value"
            >
              <td>
                Critical Hit Chance
              </td>
              <td>
                +
                 
                6
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemOffAttrTwoMax40Value"
            >
              <td>
                Critical Hit Damage
              </td>
              <td>
                +
                 
                12
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemOffAttrThreeMax40Value"
            >
              <td>
                Headshot Damage
              </td>
              <td>
                +
                 
                20
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemOffAttrFourMax40Value"
            >
              <td>
                Weapon Handling
              </td>
              <td>
                +
                 
                8
                 
                %
              </td>
            </tr>
          </tbody>
        </table>
        <table>
          <thead
            class="single-gear-item__title-bar"
          >
            <tr>
              <th
                class="single-gear-item__h3subheading"
              >
                Defensive Minor Attributes
              </th>
            </tr>
          </thead>
          <tbody>
            <tr
              data-testid="itemDefAttrOneMax40Value"
            >
              <td>
                Armour Regeneration
              </td>
              <td>
                +
                 
                4,925
                 
                per second
              </td>
            </tr>
            <tr
              data-testid="itemDefAttrTwoMax40Value"
            >
              <td>
                Explosive Resistance
              </td>
              <td>
                +
                 
                10
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemDefAttrThreeMax40Value"
            >
              <td>
                Hazard Protection
              </td>
              <td>
                +
                 
                10
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemDefAttrFourMax40Value"
            >
              <td>
                Health
              </td>
              <td>
                +
                 
                18,935
              </td>
            </tr>
          </tbody>
        </table>
        <table>
          <thead
            class="single-gear-item__title-bar"
          >
            <tr>
              <td
                class="single-gear-item__h3subheading"
              >
                Utility Minor Attributes
              </td>
            </tr>
          </thead>
          <tbody>
            <tr
              data-testid="itemUtilAttrOneMax40Value"
            >
              <td>
                Skill Damage
              </td>
              <td>
                +
                 
                10
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemUtilAttrTwoMax40Value"
            >
              <td>
                Skill Haste
              </td>
              <td>
                +
                 
                12
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemUtilAttrThreeMax40Value"
            >
              <td>
                Skill Repair
              </td>
              <td>
                +
                 
                20
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemUtilAttrFourMax40Value"
            >
              <td>
                Status Effects
              </td>
              <td>
                +
                 
                10
                 
                %
              </td>
            </tr>
          </tbody>
        </table>
      </div>
      <div
        class="single-gear-item__details"
      >
        <table
          data-testid="singleYellowArmourPieceWT5"
        >
          <thead>
            <tr
              class="single-gear-item__title-bar"
            >
              <th
                class="single-gear-item__subheading"
              >
                World Tier 5 Stats
              </th>
            </tr>
            <tr
              class="single-gear-item__title-bar"
            >
              <th
                class="single-gear-item__subheading"
              >
                Core Attributes
              </th>
            </tr>
            <tr>
              <td />
              <td>
                Max Value
              </td>
            </tr>
          </thead>
          <tbody>
            <tr
              data-testid="itemCoreOneTypeWT5"
            >
              <td>
                Weapon Damage
              </td>
              <td>
                +
                 
                9.9
                %
              </td>
            </tr>
          </tbody>
        </table>
        <table>
          <thead
            class="single-gear-item__title-bar"
          >
            <tr>
              <th
                class="single-gear-item__h3subheading"
              >
                Offensive Minor Attributes
              </th>
            </tr>
          </thead>
          <tbody>
            <tr
              data-testid="itemOffAttrOneMaxWT5Value"
            >
              <td>
                Critical Hit Chance
              </td>
              <td>
                +
                 
                5.1
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemOffAttrTwoMaxWT5Value"
            >
              <td>
                Critical Hit Damage
              </td>
              <td>
                +
                 
                7.9
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemOffAttrThreeMaxWT5Value"
            >
              <td>
                Headshot Damage
              </td>
              <td>
                +
                 
                6.9
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemOffAttrFourMaxWT5Value"
            >
              <td>
                Weapon Handling
              </td>
              <td>
                +
                 
                7
                 
                %
              </td>
            </tr>
          </tbody>
        </table>
        <table>
          <thead
            class="single-gear-item__title-bar"
          >
            <tr>
              <th
                class="single-gear-item__h3subheading"
              >
                Defensive Minor Attributes
              </th>
            </tr>
          </thead>
          <tbody>
            <tr
              data-testid="itemDefAttrOneMaxWT5Value"
            >
              <td>
                Armour Regeneration
              </td>
              <td>
                +
                 
                1,223
                 
                per second
              </td>
            </tr>
            <tr
              data-testid="itemDefAttrTwoMaxWT5Value"
            >
              <td>
                Explosive Resistance
              </td>
              <td>
                +
                 
                6.6
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemDefAttrThreeMaxWT5Value"
            >
              <td>
                Hazard Protection
              </td>
              <td>
                +
                 
                6.6
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemDefAttrFourMaxWT5Value"
            >
              <td>
                Health
              </td>
              <td>
                +
                 
                4,695
              </td>
            </tr>
          </tbody>
        </table>
        <table>
          <thead
            class="single-gear-item__title-bar"
          >
            <tr>
              <td
                class="single-gear-item__h3subheading"
              >
                Utility Minor Attributes
              </td>
            </tr>
          </thead>
          <tbody>
            <tr
              data-testid="itemUtilAttrOneMaxWT5Value"
            >
              <td>
                Skill Damage
              </td>
              <td>
                +
                 
                6.9
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemUtilAttrTwoMaxWT5Value"
            >
              <td>
                Skill Haste
              </td>
              <td>
                +
                 
                7.9
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemUtilAttrThreeMaxWT5Value"
            >
              <td>
                Skill Repair
              </td>
              <td>
                +
                 
                12.1
                 
                %
              </td>
            </tr>
            <tr
              data-testid="itemUtilAttrFourMaxWT5Value"
            >
              <td>
                Status Effects
              </td>
              <td>
                +
                 
                6.6
                 
                %
              </td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
    <div
      class="single-gear-item__title-bar"
    >
      <h3
        class="single-gear-item__h3subheading"
      >
        Number of Mod Slots
      </h3>
    </div>
    <ul>
      <li>
        1
      </li>
    </ul>
    <div
      class="single-gear-item__sub-content"
      data-testid="uniqueAttributeTest"
    >
      <div
        class="single-gear-item__details"
      >
        <div
          class="single-gear-item__title-bar"
        >
          <h3
            class="single-gear-item__h3subheading"
          >
            Item Unique Feature
          </h3>
        </div>
        <ul>
          <li>
            20% Headshot Damage
          </li>
        </ul>
      </div>
    </div>
    <div
      class="single-gear-item__title-bar"
    >
      <h2
        class="single-gear-item__subheading"
      >
        Item Unique Talent
      </h2>
    </div>
    <div
      class="single-gear-item__details"
    >
      <div
        class="LinkSmallBrand__LinkStyles-sc-1sivfgj-0 ljFRYs"
      >
        <h3
          class="LinkSmallBrand__TitleLink-sc-1sivfgj-1 gKRdRa"
        >
          <a
            href="/talents/armourTalent/%20rdvuty23684"
          >
            Perfect Intimidate
          </a>
        </h3>
        <img
          class="standard-item"
          src="FakeArmourTalentImage.jpg"
        />
      </div>
    </div>
    <div
      class="single-gear-item__title-bar"
    >
      <h2
        class="single-gear-item__subheading"
      >
        Item Brand
      </h2>
    </div>
    <div
      class="single-gear-item__details"
    >
      <div
        class="LinkSmallBrand__LinkStyles-sc-1sivfgj-0 ljFRYs"
      >
        <h3
          class="LinkSmallBrand__TitleLink-sc-1sivfgj-1 gKRdRa"
        >
          <a
            href="/gear/brand/nlkbhblhb"
          >
            Golan Gear Ltd
          </a>
        </h3>
        <img
          alt="Golan Gear Ltd"
          class="standard-item"
          src="FakeBrandImage.jpg"
        />
      </div>
    </div>
  </div>
</div>
`;
